### ISSUE TITLE
[feature] API Endpoint Documentation

### ISSUE THREAD

**Issue by Merith-TK** [skiff-org/skiff-apps#98]
I was wondering if it would be possible for `api.skiff.com` to be documented so that we can make our own clients for emails, which would allow #97 to be created by the community without effort from Skiff Developers

if the API were to be documented, I would make an "alias-server" for skiff, akin to the process described in the issue linked above, where the "middleman" will login as the designated user, and only listen for connects from an specific address, (sender address, not receiver address, so that only an specific device on the network is allowed to even send data to it for security reasons), and then "serve" an collection of "logins" for each alias the account has, 
For example

`noreply@domain.tld` would be assigned an "login" to be used by clients to connect to the server to get the emails from that address and send emails from that address, 

basically isolated logins for aliases

**Comment by CanRau** [comment]
Also looking for a public API to allow for custom workflows/filtering/labeling etc :nerd
